### PR TITLE
Add schema_search_path capability to database.yml.erb

### PR DESCRIPTION
**Pull request by justinjams**

### CODE REVIEW

**Comment by mitchless** [comment]
:+1:

File: rails/templates/default/database.yml.erb [hunk 1]
```diff
@@ -13,5 +13,8 @@
 <% if @database[:port] -%>
   port: <%= @database[:port].to_i.inspect %>
 <% end -%>
+<% if @database[:schema_search_path] -%>
+  schema_search_path: <%= @database[:schema_search_path].to_s.inspect %>
```
Comment (ncullen) [inline rails/templates/default/database.yml.erb:17]: So I assume you want to convert the whatever-type to a String, and then output the string with double-quotes around it (hence the need for `.inspect`), right?

Comment (johnlauck) [inline rails/templates/default/database.yml.erb:17]: Any reason not to use `"<%= @database[:schema_search_path] %>"`?

Comment (ncullen) [inline rails/templates/default/database.yml.erb:17]: The .to_s handles the nil case by converting the string to an empty string.  The .inspect handles quoting, so confusing characters don't end up in the yml.  Likely unnecessary 99% of the time, but that 1% means :boom: 

I _think_ a simpler equivalent is `"#{@database[:schema_search_path]}"` but perhaps the thinking is that this _always_ allocates a new string, whereas .to_s only allocates a new string if the source type isn't a string.  So `.to_s.inspect` is likely more efficient by some metric that doesn't matter.

**Pull request merged**